CODEOWNERS

```diff
@@ -76,6 +76,7 @@ homeassistant/components/essent/* @TheLastProject
 homeassistant/components/evohome/* @zxdavb
 homeassistant/components/file/* @fabaff
 homeassistant/components/filter/* @dgomes
+homeassistant/components/firmata/* @DaAwesomeP @balloob @kane610 @andrewsayre
```

Comment:
You are the code owner. Don't add all the people who participated in one way or another.